tests/core/full_node/test_mempool.py

```diff
@@ -2102,7 +2102,7 @@ def test_create_coin_with_hint(self, softfork_height):
         ],
     )
     def test_unknown_condition(self, mempool: bool, height: uint32):
-        for c in ['(1 100 "foo" "bar")', "(100)", "(1 1) (2 2) (3 3)", '("foobar")']:
+        for c in ['(2 100 "foo" "bar")', "(100)", "(3 1) (2 2) (3 3)", '("foobar")']:
```

Comment:
You might change the second `1` to a `4` since you already have a `3`. But it almost certainly doesn't matter.